File: src/Character.cs

using UnityEngine;
using UnityEngine.Events;

public class Character : MonoBehaviour
{
    public int id { get; private set; }

    public Tile startPosition { get; set; }
    //  public Tile TargetPosition;
    public Tile currentPosition;

    public bool isBot;
    public bool isMummy;
}

File: src/Character.ts
import { Tile } from "./Tile";
export class Character {
  id: number;
  startPosition: Tile;
  currentPosition: Tile;
  isMummy: boolean;
  isBot: boolean;
  GoToStartPosition(): void {
    this.currentPosition = this.startPosition;
  }
}
